src/components/ActionComplete.vue

```diff
@@ -135,7 +135,7 @@ export default {
       },
       createCampaignLetter(sessionId) {
         // Creates campaign letter with lob api.
-        let userCustomization = this.$store.state.userCustomization
+        let userCustomization = this.$store.getters.userCustomization
```

Comment:
This is correct, but for consistency, I think it would be better to use computed properties to get state data. Check out how some of the other ones are written in this file.